app/controllers/user_skills_controller.rb

```diff
@@ -12,6 +21,21 @@ def create
     end
   end
 
+  def edit
+    @user_skill = UserSkill.find(params[:id])
+  end
+
+  def destroy
+    @user_skill = UserSkill.find(params[:id])
+    @user_skill.destroy
+    redirect_to user_show_path
+  end
+
+  def update
+    @user_skill = UserSkill.find(params[:id])
+    @user_skill.update(user_skills_params)
+  end
+
   private
 
   def user_skills_params
```

Comment:
please populate this method :-)
```suggestion
  def user_skills_params
  $REQUIRE/PERMIT
  end
```